examples/resources/bloxone_dns_forward_zone/resource.tf

```diff
@@ -0,0 +1,15 @@
+resource "bloxone_dns_view" "example_view" {
+  name = "example_dns_view"
+}
+
+resource "bloxone_dns_forward_zone" "example" {
+  fqdn = "tf-acc-test.com."
+
+  # Other optional fields
+  name    = "example_forward_zone"
```

Comment:
I don't think there is a name attribute in forward zone?

Reply:
yeah no name is there . This field is removed